src/utils.tsx

```diff
@@ -0,0 +1,36 @@
+
+interface ArtData {
+  title: String,
+  artistDisplayName: String, 
+  objectDate: String | Number, 
+  primaryImage?: String, 
+  primaryImageSmall?: String
+}
+
+// Clean up data function: 
+
+const cleanUpData = (data: ArtData) => {
+  const { title, artistDisplayName, objectDate, primaryImage, primaryImageSmall } = data;
+
+  const updatedTitle = title === '' ? "Title unknown" : title
+
+  const updatedArtistName = artistDisplayName === '' ? 'Artist unknown' : artistDisplayName;
+
+  return {
+    title: updatedTitle, 
+    artistDisplayName: updatedArtistName,
+    objectDate,
+    primaryImage: primaryImage || primaryImageSmall
+  }
+}
+
+// Generate randomID: 
+
+export const generateID = (): number => Math.floor(Math.random() * 1000) + 1;
+
+export let objectID: number = generateID()
+
+// (( We don't have to use the objectID variable, we can always call the generateID function within the API call itself, or move this variable ))
+
+
+export default cleanUpData 
```

Comment:
Instead of using a combination of export and export default, we could possible do 
`export {cleanUpData, generateID, objectID }` all in one go if you want!